src/wp-admin/css/customize-controls.css

```diff
@@ -105,6 +105,12 @@ body:not(.ready) #customize-save-button-wrapper .save {
 	height: 100%;
 }
 
+@media screen and (prefers-reduced-motion: reduce) {
```

Comment:
Just wanted to double-check, as I'm not very CSS knowledgable.

The other examples https://github.com/WordPress/wordpress-develop/search?q=prefers-reduced-motion&type=code use `@media (prefers-reduced-motion: reduce)`  or `@media (prefers-reduced-motion)` (in Twenty Twenty and Twenty Twenty-One), both without specifying `screen`.

I'm wondering if it that means it should apply to contexts other than `screen` as well. What do you think?

Reply:
Yeah, I don't think it matters, but couldn't hurt to remove it :)